backend/lib/endpoints/posts.js

```diff
@@ -73,13 +73,41 @@ router.post(
     req.body.authorId = req.user.id;
     req.body.postId = postId;
     new Comment(req.body).save().then((comment) => {
```

Comment:
yes good point, I wasn't really doing strict reviews but it's better to use async/await as much as possible.
Normally also Express is using the callback pattern, I like koa more because it's made for async/await by default... I am looking at some other routes right now where it's a bit confusing if the middleware should reject or use a callback